app/services/metrics/review_turnaround/per_user_project.rb

```diff
@@ -36,8 +57,12 @@ def calculate_turnaround(review)
         review.opened_at.to_i - review.review_request.created_at.to_i
       end
 
-      def create_metric(entity, turnaround)
-        Metric.create!(ownable: entity, value: turnaround, name: :review_turnaround)
+      def create_or_update_metric(entity, turnaround)
+        metric = Metric.find_or_initialize_by(ownable: entity,
+                                              created_at: Time.zone.today.all_day)
```

Comment:
We have to add `name: 'review_turnaround'` to the query, otherwise we could be updating a different metric from the same entity.

Reply:
Yes! I forgot